### DISCUSSION TITLE
Home Assistant integration (cards) still available when Scrypted is running in another VM in Proxmox?

### DISCUSSION THREAD

**Discussion by kersten-lohmeyer** [koush/scrypted#1589] [Q&A] [answered]
Hi, first I had Scrypted running as an addon in HA but now I switched to a Proxmox VM. Seems to be running better. I was wondering if I still can use the iframe cards in HA like I was when I had Scrypted installed as an addon?

I'm kind of stuck because this was gone in HA (when running as addon and installing the HACS component):
<img width="388" alt="image" src="https://github.com/user-attachments/assets/b8e0543b-c729-418d-a004-767791122320">

... and showed the path I had to use for the iFrame. How do I do this when Scrypted runs in an extra VM?

Thanks!

**Comment by koush** [comment, marked as answer]
see docs.scrypted.app

**Reply by kersten-lohmeyer** [reply]
Thanks, was a bit confused, working now. Can I ask one more thing without opening a new post?

<img width="208" alt="Bildschirmfoto 2024-09-24 um 22 32 41" src="https://github.com/user-attachments/assets/75ee217f-eb0c-4cb0-b348-a729761f127c">

Why is Motion always unchecked and not remembered? All my events are motion based, so I always have to turn this on to see the event snapshots in the timeline. Is this also in the docs? Paying customer btw. 💪🏼😉

**Reply by koush** [reply]
you should be using object detection. the raw motion detections are not useful on the timeline since they may be nothing like trees blowing. the filters aren't saved because people disable everything on the filter and then asking why their timeline has no detections.

**Reply by kersten-lohmeyer** [reply]
Is this the right place to set this up? Which one should I choose?
<img width="729" alt="image" src="https://github.com/user-attachments/assets/a1013e59-75db-4f54-95ba-c54825e0a414">

**Reply by koush** [reply]
don't change that. it should be doing object detection by default. I'm not sure what you're doing, but if its not working see the troubleshooting in the docs. probably misconfigured motion sensor.
